shared/chat/conversation/list/index.native.js

```diff
@@ -76,7 +76,6 @@ class ConversationList extends Component<void, Props, void> {
         renderScrollComponent={this._renderScrollComponent}
         onEndReached={this._onEndReached}
         onEndReachedThreshold={0}
-        initialNumToRender={30}
```

Comment:
it shouldn't as it defaults to 10 which is more than a screenful on most devices i think